### DISCUSSION TITLE
saf-training/courses/guidance/05

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#250] [Ideas]
# saf-training/courses/guidance/05

5.1 Using Vulcan Let's walk though the process for building STIG-ready content from the beginning using the Vulcan application. We will use our demonstration instance of Vulcan ...

https://mitre.github.io/saf-training/courses/guidance/05.html

**Comment by karikarshivani** [comment]
Change demo link to staging after testing
